Exec updates aug 12 (#1910)
makerdao/community: governance/votes/Executive vote - August 12, 2022.md

- **6,628 DAI** will be transferred to [ACREInvest](https://vote.makerdao.com/address/0x4d3ac33ab1dd7b0f352b8e590fe8b62c4c39ead5) at [0x5b9C98e8A3D9Db6cd4B4B4C1F92D0A551D06F00D](https://etherscan.io/address/0x5b9C98e8A3D9Db6cd4B4B4C1F92D0A551D06F00D).
- **4,029 DAI** will be transferred to [mhonkasalo & teemulau](https://vote.makerdao.com/address/0xaa19f47e6acb02df88efa9f023f2a38412069902) at [0x97Fb39171ACd7C82c439b6158EA2F71D26ba383d](https://etherscan.io/address/0x97Fb39171ACd7C82c439b6158EA2F71D26ba383d).
- **3,797 DAI** will be transferred to [Llama](https://vote.makerdao.com/address/0x4e314eba76c3062140ad196e4ffd34485e33c5f5) at [0x82cD339Fa7d6f22242B31d5f7ea37c1B721dB9C3](https://etherscan.io/address/0x82cD339Fa7d6f22242B31d5f7ea37c1B721dB9C3).
- **2,013 DAI** will be transferred to [Blockchain@Columbia](https://vote.makerdao.com/address/0xb8df77c3bd57761bd0c55d2f873d3aa89b3da8b7) at [0xdC1F98682F4F8a5c6d54F345F448437b83f5E432](https://etherscan.io/address/0xdC1F98682F4F8a5c6d54F345F448437b83f5E432).

### HVBank (RWA-009) Legal Agreement Update

> As detailed in this [post](https://forum.makerdao.com/t/urgent-executive-inclusion-request/17179), the legal agreements for the HVBank deal will be amended in the manner described below, if this executive proposal passes. Note this item has been inserted in the place of the next 25M DAI Drawdown for HVBank, future drawdowns will be postponed until a new trading partner can step in. 

Reviewer: ```suggestion
As detailed in this [post](https://forum.makerdao.com/t/urgent-executive-inclusion-request/17179), the legal agreements for the HVBank deal will be amended in the manner described below, if this executive proposal passes. Note this item has been inserted in the place of the next 25M DAI Drawdown for HVBank. Future drawdowns will be postponed until a new trading partner steps in. 
```

> **If this Executive Proposal is passed:**

Reviewer: ```suggestion
**If this Executive Proposal passes:**
```

> The MKR token holders authorize

Reviewer: ```suggestion
`The MKR token holders authorize`
```

> The Director of RWA Foundation to instruct Ankura Trust Company, LLC to update the RWA Foundation portal with Genesis Global Trading Inc. by replacing

Reviewer: ```suggestion
`The Director of RWA Foundation to instruct Ankura Trust Company, LLC to update the RWA Foundation portal with Genesis Global Trading Inc. by replacing`
```

> 0x6C6d4Be2223B5d202263515351034861dD9aFdb6

Reviewer: ```suggestion
`0x6C6d4Be2223B5d202263515351034861dD9aFdb6`
```

> with

Reviewer: ```suggestion
`with`
```

> 0x1818EE501cd28e01E058E7C283E178E9e04a1e79

Reviewer: ```suggestion
`0x1818EE501cd28e01E058E7C283E178E9e04a1e79`
```

> The Director of RWA Foundation to instruct Genesis Global Trading Inc. to return 25 million DAI to

Reviewer: ```suggestion
`The Director of RWA Foundation to instruct Genesis Global Trading Inc. to return 25 million DAI to`
```

> 0x1818EE501cd28e01E058E7C283E178E9e04a1e79

Reviewer: ```suggestion
`0x1818EE501cd28e01E058E7C283E178E9e04a1e79`
```

> The Director of RWA Foundation to take such steps as are necessary to give effect to the above

Reviewer: ```suggestion
`The Director of RWA Foundation to take such steps as are necessary to give effect to the above`
```

> Ankura Trust Company, LLC to update the RWA Foundation portal with Genesis Global Trading Inc. to reflect the above changes to the RWA Foundation portal with Genesis Global Trading Inc.

Reviewer: ```suggestion
`Ankura Trust Company, LLC to update the RWA Foundation portal with Genesis Global Trading Inc. to reflect the above changes to the RWA Foundation portal with Genesis Global Trading Inc.`
```
